newIDE/app/src/Profile/Subscription/SubscriptionPendingDialog.js

```diff
@@ -30,11 +30,14 @@ export default function SubscriptionPendingDialog({
     !!authenticatedUser &&
     !!authenticatedUser.subscription &&
     !!authenticatedUser.subscription.planId;
-  const hasGoldOrStartupPlan =
+  const canBenefitFromDiscordRole =
```

Comment:
I reckon this should be a method shared between the 2 files for DRY purposes

Reply:
yep, you're right I'll factor that